Source/Core/Common/MemoryUtil.cpp

```diff
@@ -88,20 +88,21 @@ void FreeMemoryPages(void* ptr, size_t size)
 {
   if (ptr)
   {
-    bool error_occurred = false;
-
 #ifdef _WIN32
     if (!VirtualFree(ptr, 0, MEM_RELEASE))
-      error_occurred = true;
+    {
+      PanicAlert("FreeMemoryPages failed!\n"
+                 "VirtualFree: %s",
+                 GetLastErrorMsg().c_str());
```

Comment:
Can't this be on one line? (Same for the equivalent pieces of code in the rest of this file)